src/Mailer/Transport/AwsSesTransport.php

```diff
@@ -1,17 +1,16 @@
 <?php
 namespace CakePHP3AwsSesTransport\Mailer\Transport;
 
+use Aws\Ses\SesClient;
 use Cake\Mailer\AbstractTransport;
 use Cake\Mailer\Message;
-use Aws\Ses\SesClient;
```

Comment:
On local, @PermataTyrell created new custom plugin using this file.
Then applied `composer check` and rector for CakePHP and PHP on it.

The changes on this file are mostly to resolve errors from `composer check` result.
While the changes from rector were minimal.
Which means the original logic remains unchanged.